detectors/aws/ecs/ecs.go

```diff
@@ -43,27 +43,32 @@ var (
 )
 
 // Create interface for methods needing to be mocked
-type detectorUtils interface {
+type detectorUtilsResources interface {
```

Comment:
This rename is not related to the nil reference bug fix. Please leave them the way they were.